UnitTests/BraintreePayPalNativeCheckoutTests/BTPayPalNativeCheckoutProvider_Tests.swift

```diff
@@ -0,0 +1,45 @@
+import XCTest
+import UIKit
+import PayPalCheckout
+@testable import BraintreePayPalNativeCheckout
+
+class BTPayPalNativeCheckoutProvider_Tests: XCTestCase {
+
+    func testStart_whenStartIsCalled_checkoutIsInitialized() {
+        let nativeCheckoutProvider = BTPayPalNativeCheckoutProvider(MockCheckout.self)
+        nativeCheckoutProvider.start(
+            request: BTPayPalNativeVaultRequest(),
+            order: BTPayPalNativeOrder(payPalClientID: "", environment: .sandbox, orderID: ""),
+            nxoConfig: CheckoutConfig(clientID: ""),
+            onStartableApprove: { _, _ in },
+            onStartableCancel: { },
+            onStartableError: { _ in }
+        )
+
+        XCTAssertTrue(MockCheckout.startInvoked)
+        XCTAssertTrue(MockCheckout.isConfigSet)
+        XCTAssertFalse(MockCheckout.showsExitAlert)
```

Comment:
Would this ever be true? Doesn't look like it's changed at all

Reply:
It's mainly tested here since it's a get/set property in the `BTPayPalNativeCheckoutProtocol`, but I agree it's a bit odd since we are setting it to false a few lines below. We have it in the [PPCP test file counterpart](https://github.com/paypal/paypal-ios/blob/a0ec6196b262f51b2f7a0a68f542d64a217d85eb/UnitTests/PayPalNativePaymentsTests/NativeCheckoutProvider_Tests.swift#L22) so I added it here but feel very lukewarm on if it's necessary in the test.

Reply:
Since it isn't testing actual source code, I would agree with can remove it.